core/rio/nquads/src/test/java/org/eclipse/rdf4j/rio/nquads/AbstractNQuadsParserUnitTest.java

```diff
@@ -35,9 +35,9 @@
 import org.eclipse.rdf4j.rio.RDFParser;
 import org.eclipse.rdf4j.rio.helpers.AbstractRDFHandler;
 import org.eclipse.rdf4j.rio.helpers.BasicParserSettings;
+import org.eclipse.rdf4j.rio.helpers.NTriplesParserSettings;
```

Comment:
Is this intentional? I thought we were just reverting the JSON-LD changes.

Reply:
Yeah. All the tests are reverted so that I could be sure that there wasn't anything I had missed.

Reply:
Oh, that makes sense, thanks!